utils/socket.h

```diff
@@ -5,13 +5,13 @@
 
 #define MCOUNT_AGENT_SOCKET_DIR "/tmp/uftrace"
 
-enum uftrace_dopt;
+enum agent_opt;
```

Comment:
Please keep `uftrace` prefix for non-static functions or data structures.  Also I think 'message' would be a better term than 'option'.  How about renaming it to `enum uftrace_agent_msg`?

Reply:
Sounds good. Should I use a prefix for the enum members too? Like `UFTRACE_AGENT_MSG_CLOSE`? I'm afraid the names are getting too long.

Reply:
What about `UAM_CLOSE`?  UAM for uftrace agent message. :)

Reply:
I'm good with it.